modules/oe_theme_helper/src/Element/InPageNavigation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\Element;
+
+use Drupal\Core\Render\Element\RenderElement;
+
+/**
+ * Provides a render element for an in-page navigation element.
+ *
```

Comment:
Let's also document what render array parameters this render element accepts.

Reply:
It is simple render element without any special input parameters. Standard parameters are used: `#process`, `#pre_render`. Not sure that we should document it.